src/Bookings/beta/readme.md

```diff
@@ -30,4 +30,18 @@ require:
 title: $(service-name)
 subject-prefix: 'Beta'
 namespace: Microsoft.Graph.Beta.PowerShell
+
+directive:
+  - where:
+      verb: Get
+      subject: ^(BookingBusinessStaffAvailability)$
```

Comment:
Isn't this a valid API? See https://learn.microsoft.com/en-us/graph/api/bookingbusiness-getstaffavailability?view=graph-rest-1.0&tabs=http

Reply:
Yes, it is a valid API. However, there are two paths in the [openAPI](https://graphexplorerapi.azurewebsites.net/$openapi?tags=^solutions\.|^bookingBusinesses\.|^bookingCurrencies\.&title=Bookings&openapiversion=3&style=Powershell&fileName=powershell_v2&graphVersion=beta&forceRefresh=true&singularizeOperationIds=true) file causing a conflict  (duplicate parameter types in the body parameter) during cmdlet generation (See Autorest exception below). 

![image](https://github.com/microsoftgraph/msgraph-sdk-powershell/assets/10947120/3147cf5f-62d6-4ccd-b827-3b9fd0ee98dd)

Adding that directive retained the valid one ``/solutions/bookingBusinesses/{bookingBusiness-id}/microsoft.graph.getStaffAvailability`` and skipped the non-documented one ``/bookingBusinesses/{bookingBusiness-id}/microsoft.graph.getStaffAvailability``. Below is the screenshot of the generated cmdlet

![image](https://github.com/microsoftgraph/msgraph-sdk-powershell/assets/10947120/cea8ebc9-2726-4b4b-a8b0-3b1d2dcbe806)

Reply:
Got it!